### PR TITLE
add ilo2_232

### PR DESCRIPTION
**Pull request by sshmanko**
also changed https->http for 2.31, as all other links are http

### CODE REVIEW

File: firmware.conf [hunk 1]
```diff
@@ -49,9 +49,9 @@ url = http://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p198079150
 file = ilo196.bin
 
 [ilo2]
-version = 2.31
-url = https://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p915269625/v142909/CP033889.scexe
-file = ilo2_231.bin
+version = 2.32
+url = http://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p915269625/v144949/CP034429.scexe
```
Comment (nicolasbock) [inline firmware.conf]: Why switch to `http`? The URL also works with `https`.

Comment (sshmanko) [reply firmware.conf]: but all other 100 links in the file are http - either change all to https or stick to http imo

Comment (nicolasbock) [reply firmware.conf]: Now, that's a good point. @seveas what do you say?

**Comment by seveas** [comment]
I'd rather switch all links to https. It's 2018 after all :)

**Comment by sshmanko** [comment]
pr updated

**Comment by seveas** [comment]
Thanks!

**Pull request merged**